<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link href="default.css" type="text/css" rel="stylesheet">
<script language="javascript" type="text/javascript">
function update_modtime() {
  var x = new Date(document.lastModified);
  document.getElementById("last_mod").innerHTML="Last modified: "+x;
}
</script>
<title>Chongzhi Zang</title>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
</head>
<body bgcolor="white" onload="update_modtime();">
<div style="width: 900px; margin: auto; line-height: 1.6;">
<img align="right" src="cz_large.jpg" style="width:240px">
<h2><font color="#002F6C">Chongzhi Zang, Ph.D.</font></h2>
<p>Associate Professor, Public Health Sciences,<br>
Biochemistry and Molecular Genetics,<br>
Biomedical Engineering<br>
</p><p>
<a href="http://cphg.virginia.edu/" target="_blank" style="text-decoration:none;">Center for Public Health Genomics</a><br>
<a href="http://www.virginia.edu/" target="_blank" style="text-decoration:none;">University of Virginia</a> <a href="https://med.virginia.edu/" target="_blank" style="text-decoration:none;">School of Medicine</a><br>
P. O. Box 800717, 
Charlottesville, VA 22908<br>
</p>
<p>
Office: <a href="https://www.google.com/maps/place/West+Complex,+Charlottesville,+VA+22903/" target="_blank" style="text-decoration:none;">West Complex (MSB)</a> 6131C<br>
Phone: 434-243-5397<br>
Email: <a href="mailto:zang@virginia.edu">zang@virginia.edu</a>
</p>
<p>
CV: <a href="CV_chongzhi_zang.pdf" target="_blank">download</a><br>
Lab website: <a href="https://zanglab.github.io/" target="_blank">https://zanglab.github.io/</a>
</p>
<hr>
<h3><font color="#002F6C">Education and Training</font></h3>
<p>
B.S., Physics, <a href="http://www.pku.edu.cn/" target="_blank" style="text-decoration:none;">Peking University</a>, 2005<br></p><p>
Ph.D., Physics, <a href="http://www.gwu.edu/" target="_blank" style="text-decoration:none;">The George Washington University</a>, 2010<br></p><p>
Postdoc, Biostatistics and Computational Biology, <a href="http://www.dana-farber.org/" target="_blank" style="text-decoration:none;">Dana-Farber Cancer Institute</a>, <a href="http://www.harvard.edu/" target="_blank" style="text-decoration:none;">Harvard University</a>, 2010&ndash;2016<br>
</p><hr>
<h3><font color="#002F6C">Research Interests</font></h3>
<p>Bioinformatics methodology development; Epigenetics and chromatin biology; Transcriptional regulation; Cancer genomics and epigenomics; Statistical methods for biomedical data integration; Theoretical and computational biophysics
</p><hr>
<h3><font color="#002F6C">Research Description</font></h3>
<p>My research program focuses on two aspects: 1) developing quantitative models and computational methods for analyzing high-throughput data generated from emerging genomics technologies; and 2) using innovative computational and data science approaches to study epigenetics and transcriptional regulation of gene expression in mammalian cell systems and human diseases such as cancer.
</p><p>
How gene expression is regulated in chromatin is a fundamental question in molecular biology. The transcriptional program is a major determinant of cell identity; its dysregulation is involved in many diseases, including cancer. High-throughput genomics technologies enable us to obtain massive data measuring numerous factors and elements in the genome that affect chromatin states and gene regulation. We leverage big data and conduct computational research at the intersection of functional genomics, epigenetics, and cancer biology. Several ongoing research directions include:
</p><p>
1. <i>Bioinformatics methods for emerging omics technologies</i> 
</p><p>
Modern development of life sciences has been accelerated by new technologies. Innovative analytics methods are essential for converting high-throughput experimental data into scientific knowledge. We are interested in developing innovative statistical models and algorithms for analyzing data from emerging genomics technologies. As a pioneer in next-generation sequencing (NGS) bioinformatics, we developed SICER (<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank"><i>Bioinformatics</i> 2009</a>), one of the most widely used methods for ChIP-seq data analysis. We are currently developing new methods for unbiased analysis of data from epigenomics (ATAC-seq, CUT&RUN, CUT&Tag, etc.), single-cell multi-omics, and spatial omics techniques to study gene regulation.
</p><p>
2. <i>Machine learning methods for regulatory factor prediction and multi-omics integration</i> 
</p><p>
Transcriptional regulators (TRs, including transcription factors and chromatin regulators) are key players in transcriptional regulation. Leveraging publicly available ChIP-seq data, we developed a series of machine learning-based computational methods, including MARGE (<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank"><i>Genome Res</i> 2016</a>), BART (<a href="https://academic.oup.com/bioinformatics/article/34/16/2867/4956015" target="_blank"><i>Bioinformatics</i> 2018</a>), BARTweb (<a href="https://academic.oup.com/nargab/article/3/2/lqab022/6219159" target="_blank"><i>NAR Genom Bioinform</i> 2021</a>), and BART3D (<a href="https://academic.oup.com/bioinformatics/article/37/18/3075/6171180" target="_blank"><i>Bioinformatics</i> 2021</a>), for predicting cis-regulatory profiles and functional TRs from various input data types. Integrating public omics data with the Cancer Genome Atlas (TCGA), we curated <a href="bartcancer" target="_blank">BART Cancer</a> (<a href="https://academic.oup.com/narcancer/article/3/1/zcab011/6180061" target="_blank"><i>NAR Cancer</i> 2021</a>) for modeling transcription factor activities in TCGA cancer types. We are currently developing new methods specifically for single-cell multi-omics data, and will further develop a general framework using advanced machine learning for multi-omics integration and regulatory network prediction.
</p><p>
3. <i>Data-inspired modeling for functional epigenetics and transcriptional condensation</i> 
</p><p>
Data-driven discovery has become a new paradigm of biological research. By modeling massive data available in the public domain, we can find new patterns and new relationships in biological entities that are usually unseen from individual datasets. Integrating thousands of public omics datasets, we recently identified a cancer-specific binding pattern of CTCF, an important DNA-binding protein, and characterized its function in facilitating oncogenic transcriptional activation (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02152-7" target="_blank"><i>Genome Biol</i> 2020</a>). Inspired by emerging evidence of phase separation phenomena in gene regulation (e.g., <a href="https://www.nature.com/articles/s41586-021-03903-7" target="_blank"><i>Nature</i> 2021</a>), we will develop computational models to characterize phase-separated transcriptional condensation events from multi-omics data, with the ultimate goal of better understanding molecular mechanisms of transcriptional regulation.
</p><hr>
<h3><font color="#002F6C">Selected Publications</font></h3>
<p>
Selected from 50+ journal articles. A complete publication list can be found at my <a href="https://scholar.google.com/citations?user=GV7vSDMAAAAJ" target="_blank">Google Scholar profile</a>.<br>
* indicates authors with equal contributions. # indicates co-corresponding authors. Underscored indicates lab members.</p>
<ol style="padding-left: 24px;">
<li><p><a href="https://www.nature.com/articles/s41467-022-30624-w" target="_blank">Single-cell chromatin profiling of the primitive gut tube reveals regulatory dynamics underlying lineage fate decisions</a><br>
Ryan J. Smith*, <u>Hongpan Zhang</u>*, <u>Shengen Shawn Hu</u>*, Theodora Yung, Roshane Francis, Lilian Lee, Mark W. Onaitis, Peter B. Dirks, <u>Chongzhi Zang</u><sup>#</sup>, Tae-Hee Kim<sup>#</sup>
<br>
<i><b>Nature Communications</b></i> 13, 2965 (2022)
</p></li>
<li><p><a href="https://www.nature.com/articles/s41588-022-01069-0" target="_blank">Single-nucleus chromatin accessibility profiling highlights regulatory mechanisms of coronary artery disease risk</a><br>
Adam W. Turner*, <u>Shengen Shawn Hu</u>*, Jose Verdezoto Mosquera, Wei Feng Ma, Chani J. Hodonsky, Doris Wong, Gaëlle Auguste, Yipei Song, Katia Sol-Church, Emily Farber, Soumya Kundu, Anshul Kundaje, Nicolas G. Lopez, Lijiang Ma, Saikat Kumar B. Ghosh, Suna Onengut-Gumuscu, Euan A. Ashley, Thomas Quertermous, Aloke V. Finn, Nicholas J. Leeper, Jason C. Kovacic, Johan L.M. Björkgren, <u>Chongzhi Zang</u><sup>#</sup>, Clint L. Miller<sup>#</sup>
<br>
<i><b>Nature Genetics</b></i>, doi: 10.1038/s41588-022-01069-0 (2022)
</p></li>
<li><p><a href="https://www.nature.com/articles/s41590-022-01131-3" target="_blank">Tcf1 preprograms the mobilization of glycolysis in central memory CD8<sup>+</sup> T cells during recall responses</a><br> 
Qiang Shan*, <u>Shengen Shawn Hu</u>*, Shaoqi Zhu, Xia Chen, Vladimir P. Badovinac, Weiqun Peng, <u>Chongzhi Zang</u><sup>#</sup>, Hai-Hui Xue</u><sup>#</sup>
<br>
<i><b>Nature Immunology</b></i> 23, 386&ndash;398 (2022)
</p></li>
<li><p><a href="https://www.nature.com/articles/s41586-021-03903-7" target="_blank">UTX condensation underlies its tumour-suppressive activity</a><br> 
Bi Shi*, Wei Li*, Yansu Song*, <u>Zhenjia Wang</u>*, Rui Ju, Aleksandra Ulman, Jing Hu, Francesco Palomba, Yanfang Zhao, John Philip Le, William Jarrard, David Dimoff, Michelle A. Digman, Enrico Gratton, <u>Chongzhi Zang</u>, Hao Jiang 
<br>
<i><b>Nature</b></i> 597, 726&ndash;731 (2021)
</p></li>
<li><p><a href="https://doi.org/10.1093/nargab/lqab022" target="_blank">BARTweb: a web server for transcriptional regulator association analysis</a><br> 
<u>Wenjing Ma</u>*, <u>Zhenjia Wang</u>*, <u>Yifan Zhang</u>, Neal E Magee, <u>Yayi Feng</u>, <u>Ruoyao Shi</u>, Yang Chen, <u>Chongzhi Zang</u>
<br>
<i><b>NAR Genomics and Bioinformatics</b></i> 3(2), lqab022 (2021)
</p></li>
<li><p><a href="https://doi.org/10.1093/bioinformatics/btab173" target="_blank">BART3D: Inferring transcriptional regulators associated with differential chromatin interactions from Hi-C data</a><br> 
<u>Zhenjia Wang</u>, <u>Yifan Zhang</u>, <u>Chongzhi Zang</u>
<br>
<i><b>Bioinformatics</b></i>, btab173 (2021)
</p></li>
<li><p><a href="https://academic.oup.com/narcancer/article/3/1/zcab011/6180061" target="_blank">BART Cancer: a web resource for transcriptional regulators in cancer genomes</a><br> 
<u>Zachary V. Thomas</u>, <u>Zhenjia Wang</u>, <u>Chongzhi Zang</u>
<br>
<i><b>NAR Cancer</b></i> 3, zcab011 (2021)
</p></li>
<li><p><a href="https://doi.org/10.1186/s13059-020-02152-7" target="_blank">Cancer-specific CTCF binding facilitates oncogenic transcriptional dysregulation</a><br> 
Celestia Fang*, <u>Zhenjia Wang</u>*, Cuijuan Han, Stephanie L. Safgren, Kathryn A. Helmin, Emmalee R. Adelman, Valentina Serafin, Giuseppe Basso, Kyle P. Eagen, Alexandre Gaspar-Maia, Maria E. Figueroa, Benjamin D. Singer, Aakrosh Ratan, Panagiotis Ntziachristos<sup>#</sup>, <u>Chongzhi Zang</u><sup>#</sup>
<br>
<i><b>Genome Biology</b></i> 21, 247 (2020)
</p></li>
<li><p><a href="http://journal.hep.com.cn/qb/EN/article/downloadArticleFile.do?attachType=PDF&id=28359" target="_blank">RECOGNICER: A coarse-graining approach for identifying broad domains from ChIP-seq data</a><br>
<u>Chongzhi Zang</u><sup>#</sup>, <u>Yiren Wang</u>, Weiqun Peng<sup>#</sup>
<br>
<i><b>Quantitative Biology</b></i>, <a href="https://doi.org/10.1007/s40484-020-0225-2" target="_blank">doi:10.1007/s40484-020-0225-2</a> (2020)
</p></li>
<li><p><a href="https://doi.org/10.1016/j.isci.2020.101518" target="_blank">Polyadenylation of histone H3.1 mRNA promotes cell transformation by displacing H3.3 from gene regulatory elements</a><br>
Danqi Chen*, Qiao Yi Chen*, <u>Zhenjia Wang</u>*, Yusha Zhu, Thomas Kluz, <u>Wuwei Tan</u>, Jinquan Li, Feng Wu, Lei Fang, Xiaoru Zhang, <u>Rongquan He</u>, Steven Shen, Hong Sun, <u>Chongzhi Zang</u><sup>#</sup>, Chunyuan Jin<sup>#</sup>, Max Costa<sup>#</sup>
<br>
<i><b>iScience</b></i> 23, 101518 (2020)
</p></li>
<li><p><a href="https://www.nature.com/articles/s41586-020-2493-4" target="_blank">Expanded encyclopaedias of DNA elements in the human and mouse genomes</a><br> 
The ENCODE Project Consortium (including <u>Chongzhi Zang</u>), Jill E. Moore*, Michael J. Purcaro*, Henry E. Pratt*, Charles B. Epstein*, Noam Shoresh*, Jessika Adrian*, Trupti Kawli*, Carrie A. Davis*, Alexander Dobin*, Rajinder Kaul*, Jessica Halow*, Eric L. Van Nostrand*, Peter Freese*, David U. Gorkin*, Yin Shen*, Yupeng He*, Mark Mackiewicz*, Florencia Pauli-Behn*, Brian A. Williams, Ali Mortazavi, Cheryl A. Keller, Xiao-Ou Zhang, Shaimae I. Elhajjajy, Jack Huey, Diane E. Dickel, Valentina Snetkova, Xintao Wei, Xiaofeng Wang, Juan Carlos Rivera-Mulia, Joel Rozowsky, Jing Zhang, Surya B. Chhetri, Jialing Zhang, Alec Victorsen, Kevin P. White, Axel Visel, Gene W. Yeo, Christopher B. Burge, Eric Lécuyer, David M. Gilbert, Job Dekker, John Rinn, Eric M. Mendenhall, Joseph R. Ecker, Manolis Kellis, Robert J. Klein, William S. Noble, Anshul Kundaje, Roderic Guigó, Peggy J. Farnham, J. Michael Cherry<sup>#</sup>, Richard M. Myers<sup>#</sup>, Bing Ren<sup>#</sup>, Brenton R. Graveley<sup>#</sup>, Mark B. Gerstein<sup>#</sup>, Len A. Pennacchio<sup>#</sup>, Michael P. Snyder<sup>#</sup>, Bradley E. Bernstein<sup>#</sup>, Barbara Wold<sup>#</sup>, Ross C. Hardison<sup>#</sup>, Thomas R. Gingeras<sup>#</sup>, John A. Stamatoyannopoulos<sup>#</sup>, Zhiping Weng<sup>#</sup><br>
<i><b>Nature</b></i> 583, 699&ndash;710 (2020)
</p></li>
<li><p><a href="https://www.nature.com/articles/s41467-020-14743-w" target="_blank">An integrative ENCODE resource for cancer genomics</a><br> 
Jing Zhang*, Donghoon Lee*, Vineet Dhiman*, Peng Jiang*, Jie Xu*, Patrick McGillivray*, Hongbo Yang*, Jason Liu, William Meyerson, Declan Clarke, Mengting Gu, Shantao Li, Shaoke Lou, Jinrui Xu, Lucas Lochovsky, Matthew Ung, Lijia Ma, Shan Yu, Qin Cao, Arif Harmanci, Koon-Kiu Yan, Anurag Sethi, Gamze Gürsoy, Michael Rutenberg Schoenberg, Joel Rozowsky, Jonathan Warrell, Prashant Emani, Yucheng T. Yang, Timur Galeev, Xiangmeng Kong, Shuang Liu, Xiaotong Li, Jayanth Krishnan, Yanlin Feng, Juan Carlos Rivera-Mulia, Jessica Adrian, James R Broach, Michael Bolt, Jennifer Moran, Dominic Fitzgerald, Vishnu Dileep, Tingting Liu, Shenglin Mei, Takayo Sasaki, Claudia Trevilla-Garcia, Su Wang, Yanli Wang, <u>Chongzhi Zang</u>, Daifeng Wang, Robert J. Klein, Michael Snyder, David M. Gilbert, Kevin Yip, Chao Cheng, Feng Yue<sup>#</sup>, X. Shirley Liu<sup>#</sup>, Kevin P. White<sup>#</sup>, Mark Gerstein<sup>#</sup><br>
<i><b>Nature Communications</b></i> 11, 3696 (2020)
</p></li>
<li><p><a href="https://www.nature.com/articles/s41423-020-0436-5" target="_blank">Ectopic Tcf1 expression instills a stem-like program in exhausted CD8<sup>+</sup> T cells to enhance viral and tumor immunity</a><br> 
Qiang Shan*, <u>Sheng’en Hu</u>*, Xia Chen, Derek B. Danahy, Vladimir P. Badovinac, <u>Chongzhi Zang</u><sup>#</sup>, Hai-Hui Xue<sup>#</sup><br>
<i><b>Cellular & Molecular Immunology</b></i>, doi:10.1038/s41423-020-0436-5 (2020)
</p></li>
<li><p><a href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-019-0324-3" target="_blank">Nickel induced transcriptional changes persist post exposure through epigenetic reprograming</a><br> 
Cynthia C Jose*, <u>Zhenjia Wang</u>*, Vinay Singh Tanwar, Xiaoru Zhang, <u>Chongzhi Zang</u><sup>#</sup>, Suresh Cuddapah<sup>#</sup><br>
<i><b>Epigenetics & Chromatin</b></i> 12, 75 (2019)
</p></li>
<li><p><a href="https://doi.org/10.1093/bioinformatics/btz812" target="_blank">YY1 is a cis-regulator in the organoid models of high mammographic density</a><br> 
Qingsu Cheng, Mina Khoshdeli, Bradley S. Ferguson, Kosar Jabbari, <u>Chongzhi Zang</u><sup>#</sup>, Bahram Parvin<sup>#</sup><br>
<i><b>Bioinformatics</b></i> 36, 1663&ndash;1667 (2019)
</p></li>
<li><p><a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty194/4956015" target="_blank">BART: a transcription factor prediction tool with query gene sets or epigenomic profiles</a><br> 
<u>Zhenjia Wang</u>, Mete Civelek, Clint L. Miller, Nathan C. Sheffield, Michael J. Guertin, <u>Chongzhi Zang</u><br>
<i><b>Bioinformatics</b></i> 34, 2867&ndash;2869 (2018)
</p></li>
<li><p><a href="http://cancerres.aacrjournals.org/content/77/21/e19" target="_blank">Cistrome Cancer: a web resource for integrative gene regulation modeling in cancer</a><br> 
Shenglin Mei, Clifford A. Meyer, Rongbin Zheng, Qian Qin, Qiu Wu, Peng Jiang, Bo Li, Xiaohui Shi, Binbin Wang, Jingyu Fan, <u>Celina Shih</u>, Myles Brown, <u>Chongzhi Zang</u><sup>#</sup>, X. Shirley Liu<sup>#</sup><br>
<i><b>Cancer Research</b></i> 77, e19&ndash;e22 (2017)
</p></li><li><p>
<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank">Modeling cis-regulation with a compendium of genome-wide histone H3K27ac profiles</a><br>
Su Wang*, <u>Chongzhi Zang</u>*, Tengfei Xiao, Jingyu Fan, Shenglin Mei, Qian Qin, Qiu Wu, Xujuan Li, Kexin Xu, Housheng Hansen He, Myles Brown, Clifford A. Meyer<sup>#</sup>, X. Shirley Liu<sup>#</sup><br>
<i><b>Genome Research</b></i> 26, 1417&ndash;1429 (2016)
</p></li><li><p>
<a href="http://www.nature.com/articles/srep30255" target="_blank">NF-E2, FLI1 and RUNX1 collaborate at areas of dynamic chromatin to activate transcription in mature mouse megakaryocytes</a><br>
<u>Chongzhi Zang</u>*, Annouck Luyten*, Christina Chen, X. Shirley Liu, Ramesh A. Shivdasani<br>
<i><b>Scientific Reports</b></i> 6, 30255 (2016)
</p></li><li><p>
<a href="http://www.nature.com/articles/ncomms11305" target="_blank">High-dimensional genomic data bias correction and data integration using MANCIE</a><br> 
<u>Chongzhi Zang</u>*, Tao Wang*, Ke Deng, Bo Li, Sheng’en Hu, Qian Qin, Tengfei Xiao, Shihua Zhang, Clifford A. Meyer, Housheng Hansen He, Myles Brown, Jun S. Liu, Yang Xie<sup>#</sup>, X. Shirley Liu<sup>#</sup><br>
<i><b>Nature Communications</b></i> 7, 11305 (2016)
</p></li><li><p>
<a href="http://www.nature.com/ng/journal/v47/n11/full/ng.3404.html" target="_blank">Partitioning heritability by functional annotation using genome-wide association summary statistics</a><br>
Hilary K. Finucane*<sup>#</sup>, Brendan Bulik-Sullivan*<sup>#</sup>, Alexander Gusev, Gosia Trynka, Yakir Reshef, Po-Ru Loh, Verneri Anttila, Han Xu, <u>Chongzhi Zang</u>, Kyle Farh, Stephan Ripke, Felix R. Day, ReproGen Consortium, Schizophrenia Working Group of the Psychiatric Genomics Consortium, The RACI Consortium, Shaun Purcell, Eli Stahl, Sara Lindstrom, John R. B. Perry, Yukinori Okada, Soumya Raychaudhuri, Mark J. Daly, Nick Patterson, Benjamin M. Neale<sup>#</sup>, Alkes L. Price<sup>#</sup><br>
<i><b>Nature Genetics</b></i> 47, 1228&ndash;1235 (2015)
</p></li><li><p>
<a href="http://genesdev.cshlp.org/content/28/16/1827" target="_blank">Active enhancers are delineated de novo during hematopoiesis with limited lineage fidelity among specified primary blood cells</a><br>
Annouck Luyten*, <u>Chongzhi Zang</u>*, X. Shirley Liu<sup>#</sup>, Ramesh A. Shivdasani<sup>#</sup><br>
<i><b>Genes and Development</b></i> 28, 1827&ndash;1839 (2014)
</p></li><li><p>
<a href="http://www.pnas.org/content/111/2/705.long" target="_blank">NOTCH1-RBPJ complexes drive target gene expression through dynamic interactions with superenhancers</a><br>
Hongfang Wang*, <u>Chongzhi Zang</u>*, Len Taing, Kelly Arnett, Yinling Joey Wong, Warren S. Pear, Stephen C. Blacklow, X. Shirley Liu<sup>#</sup>, Jon C. Aster<sup>#</sup><br>
<i><b>Proceedings of the National Academy of Sciences USA</b></i> 111, 715&ndash;710 (2014)
</p></li><li><p>
<a href="http://www.nature.com/nmeth/journal/v11/n1/full/nmeth.2762.html" target="_blank">Refined DNase-seq protocol and data analysis reveals intrinsic bias in transcription factor footprint identification</a><br>
Housheng Hansen He*, Clifford A. Meyer*, Sheng’en Shawn Hu*, Mei-Wei Chen, <u>Chongzhi Zang</u>, Yin Liu, Prakash K. Rao, Teng Fei, Han Xu, Henry Long<sup>#</sup>, X. Shirley Liu<sup>#</sup>, Myles Brown<sup>#</sup><br>
<i><b>Nature Methods</b></i> 11, 73&ndash;78 (2014)
</p></li><li><p>
<a href="http://science.sciencemag.org/content/329/5994/917.full" target="_blank">PTIP promotes chromatin changes critical for immunoglobulin switch recombination</a><br>
Jeremy A. Daniel, Margarida A. Santos*, Zhibin Wang*, <u>Chongzhi Zang</u>*, Mila Jankovic, Anna Gazumyan, Kristopher R. Schwab, Arito Yamane, Darius Filsuf, Young-Wook Cho, Kai Ge, Weiqun Peng, Michel C. Nussenzweig, Rafael Casellas, Gregory R. Dressler, Keji Zhao, André Nussenzweig<br>
<i><b>Science</b></i> 329, 917&ndash;923 (2010)
</p></li><li><p>
<a href="http://www.cell.com/abstract/S0092-8674(09)00841-1" target="_blank">Genome-wide mapping of HATs and HDACs reveals distinct functions in active and inactive genes</a><br>
Zhibin Wang*, <u>Chongzhi Zang</u>*, Kairong Cui*, Dustin E. Schones, Artem Barski, Weiqun Peng, Keji Zhao<br>
<i><b>Cell</b></i> 138, 1019&ndash;1031 (2009)
</p></li><li><p>
<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank">A clustering approach for identification of enriched domains from histone modification ChIP-Seq data</a><br>
<u>Chongzhi Zang</u>, Dustin E. Schones, Chen Zeng, Kairong Cui, Keji Zhao, Weiqun Peng<br>
<i><b>Bioinformatics</b></i> 25, 1952&ndash;1958 (2009)
</p></li><li><p>
<a href="http://www.nature.com/ng/journal/v41/n8/full/ng.409.html" target="_blank">H3.3/H2A.Z double variant-containing nucleosomes mark ‘nucleosome-free regions’ of active promoters and other regulatory regions</a><br>
Chunyuan Jin*, <u>Chongzhi Zang</u>*, Gang Wei, Kairong Cui, Weiqun Peng, Keji Zhao<sup>#</sup>, Gary Felsenfeld<sup>#</sup><br>
<i><b>Nature Genetics</b></i> 41, 941&ndash;945 (2009)
</p></li><li><p>
<a href="http://www.cell.com/immunity/abstract/S1074-7613(08)00555-4" target="_blank">Global mapping of H3K4me3 and H3K27me3 reveals specificity and plasticity in lineage fate determination of differentiating CD4+ T cells</a><br>
Gang Wei*, Lai Wei*, Jinfang Zhu, <u>Chongzhi Zang</u>, Jane Hu-Li, Zhengju Yao, Kairong Cui, Yuka Kanno, Tae-Young Roh, Wendy Watford, Dustin E. Schones, Weiqun Peng, Hong-wei Sun, William E. Paul, John J. O’Shea<sup>#</sup>, Keji Zhao<sup>#</sup><br>
<i><b>Immunity</b></i> 30, 155&ndash;167 (2009)
</p></li><li><p>
<a href="http://www.cell.com/cell-stem-cell/abstract/S1934-5909(08)00582-1" target="_blank">Chromatin signatures in multipotent hematopoietic stem cells indicate the fate of bivalent genes during differentiation</a><br>
Kairong Cui*, <u>Chongzhi Zang</u>*, Tae-Young Roh, Dustin E. Schones, Richard W. Childs, Weiqun Peng, Keji Zhao<br>
<i><b>Cell Stem Cell</b></i> 4, 80&ndash;93 (2009)
</p></li><li><p>
<a href="http://www.nature.com/ng/journal/v40/n7/full/ng.154.html" target="_blank">Combinatorial patterns of histone acetylations and methylations in the human genome</a><br>
Zhibin Wang*, <u>Chongzhi Zang</u>*, Jeffrey A. Rosenfeld*, Dustin E. Schones, Artem Barski, Suresh Cuddapah, Kairong Cui, Tae-Young Roh, Weiqun Peng, Michael Q. Zhang, Keji Zhao<br>
<i><b>Nature Genetics</b></i> 40, 897&ndash;903 (2008)
</p></li><li><p>
<a href="http://wulixb.iphy.ac.cn/EN/Y2006/V55/I1/299" target="_blank">Fluorescence measurement and acoustic diagnostics of plasma channels in air</a><br>
Zuo-Qiang Hao, Jie Zhang<sup>#</sup>, Jin Yu, Zhe Zhang, Jia-Yong Zhong, <u>Chong-Zhi Zang</u>, Zhan Jin, Zhao-Hua Wang, Zhi-Yi Wei<br>
<i><b>Acta Physica Sinica</b></i> 55, 299&ndash;303 (2006)
</p></li></ol>

<hr>
<h3><font color="#002F6C">Software</font></h3>
<ul style="list-style-type:square; padding-left: 24px;"><li><p>
<b><a href="https://zanglab.github.io/SICER2/" target="_blank">SICER</a></b> (<u>S</u>patial-clustering <u>I</u>dentification of <u>C</u>hIP-<u>E</u>nriched <u>R</u>egions), a ChIP-Seq data analysis method. [<a href="https://academic.oup.com/bioinformatics/article/25/15/1952/212783" target="_blank">Publication</a>]
</p></li><li><p>
<b><a href="https://CRAN.R-project.org/package=MANCIE" target="_blank">MANCIE</a></b> (<u>M</u>atrix <u>A</u>nalysis and <u>N</u>ormalization by <u>C</u>oncordant <u>I</u>nformation <u>E</u>nhancement), a computational method for high-dimensional genomic data integration.  [<a href="http://www.nature.com/articles/ncomms11305" target="_blank">Publication</a>] 
</p></li><li><p>
<b><a href="http://cistrome.org/MARGE/" target="_blank">MARGE</a></b> (<u>M</u>odel-based <u>A</u>nalysis of <u>R</u>egulation of <u>G</u>ene <u>E</u>xpression), a comprehensive computational method for inference of cis-regulation of gene expression leveraging public H3K27ac genomic profiles in human or mouse.  [<a href="http://genome.cshlp.org/content/26/10/1417" target="_blank">Publication</a>]
</p></li><li><p>
<b><a href="https://zanglab.github.io/bart/index.htm" target="_blank">BART</a></b> (<u>B</u>inding <u>A</u>nalysis for <u>R</u>egulation of <u>T</u>ranscription), a bioinformatics tool for predicting functional transcription factors (TFs) that bind at genomic cis-regulatory regions to regulate gene expression in the human or mouse genomes, given a query gene set or a ChIP-seq dataset as input.  [<a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty194/4956015" target="_blank">Publication</a>]
</p></li></ul>
<hr>
<h3><font color="#002F6C">Honors and Awards</font></h3>
<ul style="padding-left: 24px;">
<li><p><a href="https://news.virginia.edu/content/uva-honors-distinguished-researchers-virtual-awards-event" target="_blank">University of Virginia Research Excellence Award</a> (2020)
</p></li>
<li><p>NIH/NIGMS Maximizing Investigators' Research Award (MIRA) (2019&ndash;2024)
</p></li>
<li><p><a href="https://faculty.med.virginia.edu/facultyaffairs/about-us/honors/millipub-club/" target="_blank">MilliPub Club</a>, University of Virginia School of Medicine (2018)
</p></li>
<li><p>NIH/NCI Transition Career Development Award (2017&ndash;2020)
</p></li><li>
<p><a href="http://www.lls.org" target="_blank">Leukemia and Lymphoma Society</a> Fellow Award (2012&ndash;2015)
</p></li><li>
<p><a href="https://gwtoday.gwu.edu/significant-science" target="_blank">Dimitris N. Chorafas Foundation Prize</a> (2010)
</p></li></ul>
<hr>
<h3><font color="#002F6C">Recruiting</font></h3>
<p><a href="https://zanglab.github.io/" target="_blank">My lab</a> is recruiting motivated young students and scholars to work on a variety of topics in computational biology in a collaborative research team. Postdocs, graduate students, and undergraduate students are all welcome. Please contact me for any questions.
</p><p>
Prospective postdocs can find the job details and submit applications <a href="https://zanglab.github.io/position.htm" target="_blank">here</a>.
</p>
<hr>
<p style="width: 500px; margin: auto;" align="center">
<br><i>"While the art of printing is left to us science can never be retrograde; what is once acquired of real knowledge can never be lost."<br><br>
&mdash;Thomas Jefferson, 1799</i><br><br>
</p>
<hr>
<p align="right">Last modified: August 25, 2022</p>
</div>

</body></html>